lib/client/api.go

```diff
@@ -703,7 +703,9 @@ func (tc *TeleportClient) Login() error {
 
 	// generate a new keypair. the public key will be signed via proxy if our password+HOTP
 	// are legit
-	priv, pub, err := native.New().GenerateKeyPair("")
+	keygen := native.New()
+	defer keygen.Close()
+	priv, pub, err := keygen.GenerateKeyPair("")
```

Comment:
pools feature should have been opt-out by default actually, we only need it for one case - web proxy use-case, need to make a better interface for this